### PR TITLE
layout fit example

### LINKED ISSUE #1780: Allow a Stylekit "card" to be side-by-side with width=fit-content
### Description

Cards can have their width match the size of its content with #468, which is great for displaying metrics (among other things).

I'd like to position cards side-by-side, **without needing to use a layout and explicitly defining the width of each element there**.

Currently, I can use a layout and define the width:

```python
from taipy.gui import Gui, Markdown

page_md_target = Markdown(
"""
<|layout|columns=10em 10em|

<|card|part|width=fit-content|
Metric 1
|>

<|card|part|width=fit-content|
Metric 2
|>

|>
""")

if __name__ == "__main__":
    gui = Gui(page_md_target)
    gui.run(debug=True, run_browser=False, use_reloader=True, dark_mode=False)
```

Which produces:

image

I want to create this without defining a width anywhere.

### Solution Proposed

Maybe allowing a layout column to "fit-content" ("fc") is a possible solution? I don't like the extra nesting especially if I'm only showing 2 cards, but it makes sense and I can also use layout.gap to add space between the cards.

```python
page_md_layout = Markdown(
"""
<|layout|columns=fc fc|

<|card|part|width=fit-content|
Metric 1
|>

<|card|part|width=fit-content|
Metric 2
|>

|>
""")
```

### Impact of Solution

Needs to work with taipy.gui.builder / tgb also.

### Additional Context

_No response_

### Acceptance Criteria

- [ ] Ensure new code is unit tested, and check code coverage is at least 90%.
- [ ] Create related issue in taipy-doc for documentation and Release Notes.
- [ ] Check if a new demo could be provided based on this, or if legacy demos could be benefit from it.
- [ ] Ensure any change is well documented.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
resolves #1780

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
show that there's no need for a change in the code to achieve the desired result

image

## Related Tickets & Documents
- Closes #1780 

## How to reproduce the issue

```py
from taipy.gui import Gui

page = """
<|layout|columns=max-content max-content|

<|card|part|
France
|>

<|card|part|
United States of America
|>

|>
"""

if __name__ == "__main__":
    Gui(page).run(title="layout - fit content")

```

### CODE REVIEW

File: doc/gui/examples/blocks/layout_fit/markdown.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<|card|part|
+Metric 2
+|>
+
+|>
+"""
+
+if __name__ == "__main__":
+    Gui(page).run(title="layout - fit content")
```
Comment (FabienLelaquais) [inline doc/gui/examples/blocks/layout_fit/markdown.py]:
```suggestion
    Gui(page).run(title="Layout - Fit to content")
```

File: doc/gui/examples/blocks/layout_fit/builder.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+with tgb.Page() as page:
+    with tgb.layout("max-content max-content"):  # type: ignore[attr-defined]
+        with tgb.part("card"):  # type: ignore[attr-defined]
+            tgb.text("Metric 1")  # type: ignore[attr-defined]
+        with tgb.part("card"):  # type: ignore[attr-defined]
+            tgb.text("Metric 2")  # type: ignore[attr-defined]
+
+if __name__ == "__main__":
+    Gui(page).run(title="layout - fit content")
```
Comment (FabienLelaquais) [inline doc/gui/examples/blocks/layout_fit/builder.py]:
```suggestion
    Gui(page).run(title="Layout - Fit to content")
```

**Review by FabienLelaquais — DISMISSED** [review]
Samples are fine.
But... where is 'fc'??

**Review by dinhlongviolin1 — DISMISSED** [review]

**Review by FabienLelaquais — COMMENTED** [review]
👍

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**